controllers/placementapi_controller.go

```diff
@@ -591,7 +719,14 @@ func (r *PlacementAPIReconciler) reconcileNormal(ctx context.Context, instance *
 	//
 	// check for required OpenStack secret holding passwords for service/admin user and add hash to the vars map
 	//
-	ospSecret, hash, err := oko_secret.GetSecret(ctx, helper, instance.Spec.Secret, instance.Namespace)
+	hash, result, ospSecret, err := ensureSecret(
+		ctx,
+		types.NamespacedName{Namespace: instance.Namespace, Name: instance.Spec.Secret},
+		[]string{
+			instance.Spec.PasswordSelectors.Service,
```

Comment:
We use other fields from the secret too:
https://github.com/openstack-k8s-operators/placement-operator/blob/269c641305039b0f80edb948b6910c92fb3ce7d1/pkg/placement/deployment.go#L167-L168

Reply:
Thanks, done!